ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -405,7 +405,18 @@ def __init__(self, log_dir):
             raise RuntimeError("This contrib module requires tensorboardX to be installed. "
                                "Please install it with command: \n pip install tensorboardX")
 
-        self.writer = SummaryWriter(logdir=log_dir)
+        try:
+            self.writer = SummaryWriter(logdir=log_dir)
+        except TypeError:
```

Comment:
Probably we would like check if this TypeError is really related to `logdir` vs `log_dir`. Otherwise, warning can be misleading...

Reply:
Ok, I'm checking against the exact error message now so we'll only catch the one.